### PR TITLE
Fixed SQL errors if allowed IDs are empty.

### PR DESCRIPTION
**Pull request by bogdanarizancu**

### CODE REVIEW

**Review by chrisbaltazar — APPROVED** [review]
very nice 👍

File: src/CliCommand.php [hunk 1]
```diff
@@ -97,10 +97,19 @@ public function __invoke(array $args, array $options) {
...
+        $tableWheres["{$wpdb->prefix}users"] = "ID IN ({$allowedUserIds})";
+        $tableWheres["{$wpdb->prefix}usermeta"] = "user_id IN ({$allowedUserIds})";
+      }
+      if ($allowedOrderIds) {
+        $tableWheres["{$wpdb->prefix}comments"] = "comment_post_ID IN ({$allowedOrderIds})";
+        $tableWheres["{$wpdb->prefix}woocommerce_order_items"] = "order_id IN ({$allowedOrderIds})";
+      }
+      if ($allowedOrderItemIds) {
+        $tableWheres["{$wpdb->prefix}woocommerce_order_itemmeta"] = "order_item_id IN ({$allowedOrderItemIds})";
+      }
```
Comment (sun) [inline src/CliCommand.php:111]: Can we move these in the WooCommerce group/block below?

**Comment by sun** [comment]
Not sure how there can be empty user IDs though. Every site needs at least one administrator account, so this is technically not possible.

**Review by sun — APPROVED** [review]
We can move forward with this. But as mentioned before, this case most likely does not result in a functional website, because too much data is omitted in the dump. Therefore I added a warning message to inform the user about this problem.

If you are experiencing the issue of empty `$allowedUserIds` in one of our current sites, then you will have to dig deeper than this PR.  If you only faced the WooCommerce related query errors, and you only added the same condition for the user IDs for consistency in the code, then everything is okay.

**Comment by bogdanarizancu** [comment]
> We can move forward with this. But as mentioned before, this case most likely does not result in a functional website, because too much data is omitted in the dump. Therefore I added a warning message to inform the user about this problem.
> 
> If you are experiencing the issue of empty `$allowedUserIds` in one of our current sites, then you will have to dig deeper than this PR. If you only faced the WooCommerce related query errors, and you only added the same condition for the user IDs for consistency in the code, then everything is okay.

@sun — you are right, I only experienced empty order ids, so a shop with no orders yet, and thought maybe check users while at it. Now that you mentioned it, it doesn't really makes sense to check for admins, if that one is empty, you've got bigger problems than not being able to clean export your database, I will actually remove this to keep the codebase as clean as possible, thanks for the tip.

File: src/CliCommand.php (hunk 1 above)
Comment (bogdanarizancu) [reply src/CliCommand.php]: Not sure what you mean here, but we can refactor later on.

**Pull request merged**